velox/tool/trace/OperatorReplayerBase.cpp

```diff
@@ -103,4 +110,15 @@ OperatorReplayerBase::replayNodeFactory(const core::PlanNode* node) const {
     return createPlanNode(node, nodeId, source);
   };
 }
+
+void OperatorReplayerBase::printStats(
+    const std::shared_ptr<exec::Task>& task) const {
+  const auto planStats = exec::toPlanStats(task->taskStats());
+  const auto& stats = planStats.at(replayPlanNodeId_);
+  for (const auto& [name, operatorStats] : stats.operatorStats) {
+    LOG(INFO) << "Stats of replaying operator " << name << " : "
+              << operatorStats->toString();
+  }
+  LOG(INFO) << "Memory usage: " << task->pool()->treeMemoryUsage(false);
+}
```

Comment:
This does not need to be a member of OperatorReplayerBase if it is not using any members of this class. We can make it static or simply put it as a helper outside of the class.

Reply:
Yes, it uses `OperatorReplayerBase ::replayPlanNodeId_`